packages/block-library/src/missing/edit.native.js

```diff
@@ -18,23 +18,79 @@ import { __ } from '@wordpress/i18n';
  */
 import styles from './style.scss';
 
+const isAndroid = Platform.OS === 'android';
+const platformText = isAndroid ? 'Android' : 'iOS';
 export class UnsupportedBlockEdit extends Component {
+	constructor( props ) {
+		super( props );
+		this.state = { showHelp: false };
+	}
+
+	toggleSheet() {
+		this.setState( {
+			showHelp: ! this.state.showHelp,
+		} );
+	}
+
+	renderHelpIcon() {
+		return <View style={ styles.helpIconContainer } >
+			<Icon
+				className="unsupported-icon-help"
+				label={ __( 'Help icon' ) }
+				icon="editor-help"
+			/>
+		</View>;
+	}
+
+	renderSheet() {
+		const marginBottom = isAndroid ? 16 : 0;
```

Comment:
Why do we need this extra margin in Android? Maybe a comment will be good to add a comment to explain the need.

Reply:
I will put a comment there :)